src/store/components/AdminItemPage/index.tsx

```diff
@@ -22,79 +23,115 @@ interface AdminItemPageProps {
+    options: item?.hasVariantsEnabled
+      ? item?.options.map((option) => ({
+          uuid: option.uuid,
+          value: option.metadata?.value ?? '',
+          price: option.price.toString(),
+          quantity: option.quantity.toString(),
+          discountPercentage: option.discountPercentage.toString(),
+        }))
+      : [],
+    // Variants Disabled
+    quantity: (!item?.hasVariantsEnabled && item?.options[0].quantity?.toString()) || '',
+    price: (!item?.hasVariantsEnabled && item?.options[0].price.toString()) || '',
+    discountPercentage: (!item?.hasVariantsEnabled && item?.options[0].discountPercentage.toString()) || '',
+  };
+
   return (
     <>
       <StoreHeader breadcrumb breadcrumbLocation="/store" />
       <div className="admin-item-page">
         <h2 className="admin-item-page-title">{creatingItem ? 'Create Item' : 'Edit Item'}</h2>
         <Formik
           enableReinitialize
-          initialValues={{
-            collection: item?.collection.uuid,
-            itemName: item?.itemName,
-            description: item?.description,
-            existingPicture: item?.picture,
-            newPicture: null,
-            hidden: false, // TODO
-            monthlyLimit: item?.monthlyLimit.toString() ?? '',
-            lifetimeLimit: item?.lifetimeLimit.toString() ?? '',
-            hasVariantsEnabled: item?.hasVariantsEnabled,
-            // Variants Enabled
-            categoryName: item?.hasVariantsEnabled ? item?.options[0].metadata?.type : '',
-            options: item?.hasVariantsEnabled
-              ? item?.options.map((option) => ({
-                  uuid: option.uuid,
-                  value: option.metadata?.value ?? '',
-                  price: option.price.toString(),
-                  quantity: option.quantity.toString(),
-                  discountPercentage: option.discountPercentage.toString(),
-                }))
-              : [],
-            // Variants Disabled
-            quantity: !item?.hasVariantsEnabled ? item?.options[0].quantity?.toString() : '',
-            price: !item?.hasVariantsEnabled ? item?.options[0].price.toString() : '',
-            discountPercentage: !item?.hasVariantsEnabled ? item?.options[0].discountPercentage.toString() : '',
-          }}
-          validate={(values) => {
-            const errors: AdminItemPageForm = {};
-            if (!values.itemName) {
-              errors.itemName = 'Required';
-            }
-            return errors;
-          }}
+          initialValues={initialValues}
+          validationSchema={AdminItemPageFormSchema}
           onSubmit={async (values, { setSubmitting }) => {
             const url = creatingItem ? `${Config.API_URL}${Config.routes.store.item}` : `${Config.API_URL}${Config.routes.store.item}/${item?.uuid}`;
 
-            const payload = values.hasVariantsEnabled
+            const payload: any = values.hasVariantsEnabled
```

Comment:
Casting this value to `any`, so that the below `delete` line doesn't require additional typing.

Reply:
damn typescript is annoying :( 

the only way to avoid [`any` here](https://stackoverflow.com/questions/63702057/what-is-the-logic-behind-the-typescript-error-the-operand-of-a-delete-operato) is to create an entire interface for the type of payload that makes `options` optional (but that's not worth the ffort)